src/cli/tc.rs

```diff
@@ -104,11 +108,37 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
 }
 
 fn display_config(config: &Config, n_plus_one_queries: bool) {
-  Fmt::display(Fmt::success(&"No errors found".to_string()));
   let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
   Fmt::display(Fmt::table(seq));
 }
+
+fn validate_operations(blueprint: &Blueprint, operations: Vec<String>) -> Result<()> {
+  tokio::runtime::Runtime::new()?.block_on(async {
+    let schema = blueprint.to_schema();
+    let mut execution = vec![];
+
+    for op in operations.iter() {
+      if let Ok(operation) = tokio::fs::read_to_string(op).await {
+        // Fake execution to get errors
+        let Response { errors, .. } = schema.execute(&operation).await;
+        execution.push((op, errors));
+      } else {
+        return Err(anyhow!("Cannot read operation {}", op));
+      }
+    }
```

Comment:
Instead of creating a mut variable and pushing to it, we could use `.map` and assign it directly

e.g.:
```suggestion
    let mut execution = vec![];
    let execution = operations.iter()
        .map(/* the operation processing */)
```

Reply:
also, once we do this, we can map into `format_operation` directly, and on `Fmt::display`, we'll need only to `join(\n\n)` (and possibly `.collect::<Vec<String>>()`)

Reply:
But then, it will require to handle failures in reading in another way. I'll look into it. Thanks

Reply:
If you handle the failures as mentioned [here](https://github.com/tailcallhq/tailcall/pull/706/files#r1411120957) (with `unwrap_or_else`), you can do it with only a few tweaks.